src/rofi_rbw/clipboarder.py

```diff
@@ -77,10 +77,27 @@ def copy_to_clipboard(self, characters: str) -> None:
             encoding='utf-8'
         )
 
+        self.previously_copied = characters
+
+    def get_clipboard(self) -> str:
```

Comment:
Can we please rename that to something better fitting? It doesn't get the clipboard, it's about the clipboard's content. So maybe `fetch_content` or `fetch_clipboard_content`?

Reply:
Sounds good, I changed it to `fetch_clipboard_content`